app/presenters/courses_presenter.rb

```diff
@@ -151,9 +151,9 @@ def trained_percent
   end
 
   def wikidata_stats
-    stats ||= courses.joins(:course_stat).where.not(course_stats: nil).map do |course|
+    stats ||= courses.joins(:course_stat).where.not(course_stats: nil).map { |course|
       course.course_stat.stats_hash['www.wikidata.org']
-    end
+    }.compact
```

Comment:
I think rubocop will complain about using braces for a multi-line block instead of do/end. You can still chain the do/end syntax with `end.compact`.

Reply:
ohkk, thanks.